recipes/django-tables2/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "django-tables2" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jieter/django-tables2/archive/v{{ version }}.tar.gz
+  sha256: e06bf2b66016bf56ec906828842ec7f1c03c70477d3fd6c1d7341f46230cbb49
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - django >=2.2
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+    - python -c "import django; from django.conf import settings; settings.configure(); django.setup(); import django_tables2;"
+
+about:
+  home: https://github.com/jieter/django-tables2
+  license: Simplified BSD
```

Please check that the text matches but I suspect this is:
```suggestion
  license: BSD-2-Clause
```

---

It is indeed, updated